courses/machine_learning/deepdive2/structured/labs/1a_explore_data_babyweight.ipynb

```diff
@@ -13,7 +13,7 @@
     "\n",
```

Comment:
<p>"the goal is to predict..." --&gt; "the goal of our machine learning model is to predict.."</p><p> </p><p>Title this section "Use BigQuery to explore the dataset"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/946/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>